app/lib/contentdm/collection.rb

```diff
@@ -0,0 +1,31 @@
+###
+# This class creates and returns a collection object
+# It will first look for a Collection with the title it is given
+# if it finds one, it uses that collection. If it doesn't find one
+# it will create a new collection with the given title.
+module Contentdm
+  class Collection
+    ##
+    # @param title [String]
+    def initialize(title)
+      @title = title
+      @collection_id
+    end
+
+    ##
+    # @return [ActiveFedora::Base]
+    def create
```

Comment:
Can you change this method name to something like `find_or_create` since it attempts to find existing collection first.

Reply:
:+1: